client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -940,3 +940,23 @@ text.slicetext {
   }
 }
 
+.ui-select-choices-row > span {
+  background-color: inherit !important;
+}
+
+.inactive,
+.ui-select-choices-row.disabled {
+  background-color: #eee !important;
+  border-color: transparent;
+  opacity: 0.5;
+  box-shadow: none;
+  color: #333;
+  pointer-events: none;
+  cursor: not-allowed;
+}
+
+.inactive .label,
```

Comment:
@emtwo can you make this selector (also the one above) more specific? It's a bit broad and thus risky.